lib/lita/services/lunch_assigner.rb

```diff
@@ -100,6 +100,16 @@ def persist_winning_lunchers
         end
       end
 
+      def persist_wagers
+        sw = Lita::Services::SpreadsheetManager.new(ENV.fetch('GOOGLE_SP_DB_KEY'), 'wagers')
+        time = Time.now.strftime('%Y-%m-%d')
```

Comment:
Do not use Time.now without zone. Use one of Time.zone.now, Time.current, Time.now.in_time_zone, Time.now.utc, Time.now.getlocal, Time.now.xmlschema, Time.now.iso8601, Time.now.jisx0301, Time.now.rfc3339, Time.now.httpdate, Time.now.to_i, Time.now.to_f instead.